Rb.Forms.Barcode.nuspec

```diff
@@ -14,7 +14,11 @@ Please check the website for further details.
 
 Beta notice: Currently only android is supported.</description>
     <summary>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes.</summary>
-    <releaseNotes>0.4.1-beta:
+    <releaseNotes>0.4.2-beta:
+* Fixed: NullReferenceException
+* Improved: Updated to Xamarin.Forms 1.4.4.6392
```

Comment:
Added ZXing.Net.Mobile and Xamarin.Android.Support.v4 to the list of improvements